onnx/defs/nn/defs.cc

```diff
@@ -1281,6 +1281,12 @@ ONNX_OPERATOR_SET_SCHEMA(
             "var",
             "running (training) or estimated (testing) variance tensor of shape (C).",
             "T")
+        .Input(
+            5,
+            "is_train",
+            "If set to nonzero, run spatial batch normalization in training mode, default is 0.",
```

Comment:
I think is_train is type tensor(boolean). Maybe we should consider using {true, false} instead.